tools/mksurfdata_map/src/mksurfdat.F90

```diff
@@ -1561,6 +1537,7 @@ subroutine normalizencheck_landuse(ldomain)
     end do
 
     ! Check that when pctnatveg+pctcrop identically zero, sum of special landunits is identically 100%
+    ! KO - Not sure if this needs to be changed or is necessary
```

Comment:
I don't think that this error check needs to be kept.  The intent is:

! Check that when pctnatveg+pctcrop identically zero, sum of special landunits is identically 100%

If we treat crop like the special landunits, then this would be modified to:

! Check that when pctnatveg is identically zero, sum of special plus crop landunits is identically 100%

and we've already assured that in the modified statements above.

Reply:
Sounds good - that makes sense to me.

Reply:
Removed this error check.